src/commands/projects/list.ts

```diff
@@ -38,14 +54,33 @@ Use this command to list projects that are part of a particular organization.`
       id: {header: 'ProjectId'},
       displayName: {header: 'Name'},
       languageTopic: {header: 'LanguageTopic'},
+      datapacks: {
```

Comment:
Let's keep the "Channels, Datapacks" order (left to right) in the list of columns so that users who don't use the new flags get the same CSV output